### PR TITLE
fix: Alert add css word break

### LINKED ISSUE #18923: Alert 组件 文字 换行 效果有问题 看图
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://codepen.io/suyunlongsy/pen/VwZVxKQ](https://codepen.io/suyunlongsy/pen/VwZVxKQ)

### Steps to reproduce
GitHub Error

### What is expected?
正常折行 word-break break-all 

### What is actually happening?
没有正常折行

| Environment | Info |
|---|---|
| antd | 3.23.3 |
| React | codePen |
| System | codePen |
| Browser | codePen |

### PR DESCRIPTION
**Pull request by yoyo837**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix #18923

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Alert word break       |
| 🇨🇳 Chinese |   Alert 标题和内容单词折行        |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
`lint` CI failed. Please check.

File: components/alert/style/index.less [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
   position: relative;
   padding: 8px 15px 8px 37px;
+  word-wrap: break-word;
```
Comment (shaodahong) [inline components/alert/style/index.less]: 这里使用 `word-break: break-all;` 是不是更好，长单词换行感觉不美观

Comment (shaodahong) [reply components/alert/style/index.less]: image

image

Comment (yoyo837) [reply components/alert/style/index.less]: 英文拆词没影响吗？我搜了一下都是break-word;

Comment (yoyo837) [reply components/alert/style/index.less]: 例句的长单词长度太极端了

**Pull request merged**